docs/api-guide/caching.md

```diff
@@ -28,33 +28,37 @@ from rest_framework import viewsets
 
 class UserViewSet(viewsets.ViewSet):
     # With cookie: cache requested url for each user for 2 hours
-    @method_decorator(cache_page(60*60*2))
+    @method_decorator(cache_page(60 * 60 * 2))
     @method_decorator(vary_on_cookie)
     def list(self, request, format=None):
         content = {
-            'user_feed': request.user.get_user_feed()
+            "user_feed": request.user.get_user_feed(),
         }
         return Response(content)
 
 
 class ProfileView(APIView):
     # With auth: cache requested url for each user for 2 hours
-    @method_decorator(cache_page(60*60*2))
-    @method_decorator(vary_on_headers("Authorization",))
+    @method_decorator(cache_page(60 * 60 * 2))
+    @method_decorator(
+        vary_on_headers(
+            "Authorization",
+        )
+    )
```

Comment:
I think this looks better, dont you think so?

```python
@method_decorator(vary_on_headers("Authorization"))
```

Reply:
@baseplate-admin Absolutely, I didn't want this PR to touch any of this though, that's why I only applied changes that _I had to_. But will push this, it's a very minor change. Thanks!

Reply:
@baseplate-admin Pushed (squashed change into previous commit).